src/external-function-context.tsx

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
+import { createContext } from 'react';
 import type { FormField } from './types';
 
-export type ExternalFunctionContextProps = {
+/**
```

Comment:
@NethmiRodrigo does this comment string explain what this context provider does accurately?

Reply:
I made into a generic name so that it could be used for similar props in the future, but do you think it seems a bit too generic?